pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -200,7 +200,9 @@ type OFBridge struct {
 	// connected is an internal channel to notify if connected to the OFSwitch or not. It is used only in Connect method.
 	connected chan bool
 	// pktConsumers is a map from PacketIn category to the channel that is used to publish the PacketIn message.
-	pktConsumers      sync.Map
+	pktConsumers sync.Map
+
+	mpReplyChsMutex   sync.RWMutex
 	multipartReplyChs map[uint32]chan *openflow15.MultipartReply
```

Comment:
maybe rename this to `mpReplyChs` for consistency?

Reply:
Sure, will update.